js_modules/dagster-ui/packages/ui-core/src/instance/backfill/BackfillOpJobPartitionsTable.tsx

```diff
@@ -0,0 +1,80 @@
+import {Box, NonIdealState, Table} from '@dagster-io/ui-components';
+import {useMemo} from 'react';
+
+import {StatusBar} from './BackfillAssetPartitionsTable';
+import {BackfillTarget} from './BackfillRow';
+import {BackfillDetailsBackfillFragment} from './types/useBackfillDetailsQuery.types';
+import {failedStatuses, inProgressStatuses, successStatuses} from '../../runs/RunStatuses';
+import {numberFormatter} from '../../ui/formatters';
+import {buildRepoAddress} from '../../workspace/buildRepoAddress';
+
+export const BackfillOpJobPartitionsTable = ({
+  backfill,
+}: {
+  backfill: BackfillDetailsBackfillFragment;
+}) => {
+  const results = useMemo(() => backfill.partitionStatuses?.results || [], [backfill]);
+
+  const repoAddress = backfill.partitionSet
+    ? buildRepoAddress(
+        backfill.partitionSet.repositoryOrigin.repositoryName,
+        backfill.partitionSet.repositoryOrigin.repositoryLocationName,
+      )
+    : null;
+
+  const inProgress = useMemo(
+    () => results.filter((result) => inProgressStatuses.has(result.runStatus as any)),
+    [results],
+  );
+  const succeeded = useMemo(
+    () => results.filter((result) => successStatuses.has(result.runStatus as any)),
+    [results],
+  );
+  const failed = useMemo(
+    () => results.filter((result) => failedStatuses.has(result.runStatus as any)),
+    [results],
+  );
+
+  if (!backfill.partitionStatuses) {
+    return (
+      <Box margin={48}>
+        <NonIdealState
+          title="Partition statuses unavailable"
+          description="Dagster was unable to load per-partition statuses. This may occur if the backfilled assets or jobs no longer exist in your loaded code locations."
```

Comment:
I think we don't normally refer to Dagster in the third person. Maybe just "Unable to load per-partition statuses"?